### DISCUSSION TITLE
AVM Zwangsupdate auf neue Firmware verhindern?

### DISCUSSION THREAD

**Discussion by AkteX** [Freetz-NG/freetz-ng#1033] [Allgemein] [answered]
AVM hat einen automatischen Update auf die neue Firmware angekündigt. Kann man dies irgendwie verhindern?

Für Ihre FRITZ!Box 7520 ist das neue FRITZ!OS 7.59 verfügbar.
Aktuell verwenden Sie FRITZ!OS-Version 7.57.
Das Update auf das neue FRITZ!OS erfolgt automatisch und startet voraussichtlich am 22.08.2024 ab 03:50 Uhr.

**Comment by aus-sua** [comment, marked as answer]
Wenn AVM nichts Grundlegendes geändert hat, ist die Lösung doch seit 15 Jahren bekannt/ähnlich...

1.  System --> Update --> Auto-Update:
auf "Stufe I: Über neue FRITZ!OS-Versionen informieren" stellen
und
"Updates über andere Heimnetzgeräte anstoßen" abwählen
und
"Übernehmen" anklicken
![_1 _](https://github.com/user-attachments/assets/60aaf0f1-bcfa-44b6-9bba-81e8778af821)

2.  Internet --> Zugangsdaten --> AVM-Dienste:
"FRITZ!Box sucht periodisch nach Updates" abwählen
und
"Übernehmen" anklicken
![_2 _](https://github.com/user-attachments/assets/51be08f6-e6ce-496e-b2ec-35b6f8f7a680)

 2.1.  System --> Update --> Auto-Update:
  kontrollieren, auf "ausge-grau-t"
![_2 1 _](https://github.com/user-attachments/assets/ce05468e-5068-49db-978d-51e64198a0df)


**3.  ganz wichtig, unbedingt (!) :**
System --> Sicherung --> Neustart:
"Neu starten" anklicken!
Oder eben für 20 Sekunden Netzteilstecker ziehen...
![_3 _](https://github.com/user-attachments/assets/3b78d337-7805-4f0e-9652-89c4d9dc5d9e)



Wenn Du freetz-NG benutzt, könntest Du bei der Image-Erstellung, unter "Removal patches":
[x] Remove public firmware key
auswählen.
Das ist der harte Weg, da macht die Box zwar das Update (wenn es noch aktiviert ist), aber 
ohne AVM-firmware-key kann es das nicht installieren...
![_freetz-ng_](https://github.com/user-attachments/assets/b005bbc7-b0da-4035-98e9-d5ace0a1ceb9)
